basics/upload.md

```diff
@@ -0,0 +1,45 @@
+# Uploading a photo
+To upload a photo, you need to do 2 requests:
+ 1. `POST /photos`
+ 1. `POST url-obtained-from-first-request`
+ 
+First, send a `POST` request to `/photos` ([see documentation](https://github.com/500px/api-documentation/blob/master/endpoints/photo/POST_photos.md)) with the photo details like title, description and so on. You may also want to set the `privacy` parameter to `1` if you wish to do a private upload or `0` for public upload.
```

Comment:
I might add a note about the `auto_activate` parameter here too, as it's important to make sure that it's set.

Reply:
@mwstobo done!